Blizzard.IO.RabbitMQ.Tests/Rpc/ConcreteMessageSerializationStrategyTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Linq;
+using ISerializer = Blizzard.IO.Core.Rpc.ISerializer;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    [TestFixture]
+    public class ConcreteMessageSerializationStrategyTests
+    {
+        private Mock<ISerializer> _serializerMock;
+        private ConcreteMessageSerializationStrategy _concreteMessageSerializationStrategy;
+
+        [SetUp]
+        public void Setup()
+        {
+            _serializerMock = new Mock<ISerializer>();
+            _concreteMessageSerializationStrategy = new ConcreteMessageSerializationStrategy(_serializerMock.Object);
+        }
+
+        [Test]
+        public void SerializeMessage_OnValidIMessage_ShouldReturnsValidSerializedMessage()
+        {
+            //Arrange
+            long data = 1000;
+
+            byte[] serializedData = new byte[]
+            {
+                0,1,2,3,4,5
+            };
+
+            _serializerMock.Setup(sr => sr.Serialize(It.IsAny<long>())).Returns(serializedData);
+            MessageProperties messageProperties = new MessageProperties();
+
+
+            IMessage message = new Message<long>(data, messageProperties);
+
+            //Act
+            SerializedMessage output = _concreteMessageSerializationStrategy.SerializeMessage(message);
+
+            //Assert
+            Assert.True(serializedData.SequenceEqual(output.Body));
+            Assert.True(output.Properties == messageProperties);
+        }
```

Comment:
Add an extra test for `SerializeMessage` verifying that `_serializer` serializes the `body` param

Reply:
f